.github/workflows/csharp.yml

```diff
@@ -32,7 +32,7 @@ concurrency:
 
 jobs:
     run-tests:
-        timeout-minutes: 15
+        timeout-minutes: 25
```

Comment:
Didn't want to put this change in this PR, but I couldn't split it out for obvious reasons.